### ISSUE TITLE
Two best comments on the mikroblog

### ISSUE THREAD

**Issue by km4lin** [feelfreelinux/WykopMobilny#129]
It would be awesome to see a two best comments on the mikroblog like it is on web browser.
Regards

**Comment by rnickson** [comment]
I don't think it's a good idea, because this, if implemented, will strongly lower app performance, as this will require app to check comments to every entry with new request to Wykop API.

**Comment by feelfreelinux** [comment]
Yeah, it's not quite possible unless Wykop provides better api. But sorting comments via vote count should be possible / or highlighting two most voted comments

**Comment by km4lin** [comment]
Oh, so if it depends on the better api I suppose that it will be possible somehow... never
